lib/classes/Variables.js

```diff
@@ -516,7 +517,9 @@ class Variables {
     if (this.tracker.contains(variableString)) {
       ret = this.tracker.get(variableString, propertyString);
     } else {
-      if (variableString.match(this.envRefSyntax)) {
+      if (variableString.match(this.slsRefSyntax)) {
+        ret = this.getValueFromSls(variableString);
```

Comment:
Hmm.. what about adding a special case to `self:` like we already have for `self:provider` for example? https://github.com/serverless/serverless/blob/9a0513b007af2c50659e7f18c339ed25d8f94b5b/lib/classes/Variables.js#L602

Reply:
Yes, I was also thinking about that. I considered `${self:}` and `${env:}`. The problem with `self` is that it's always referencing the `serverless.yml` (at least that's the mental model).

So I thought it would be confusing to add it and it's not visible in the `yml`.

But good feedback :+1: